components/AddContentCreatorForm/FormStatus.tsx

```diff
@@ -38,6 +38,16 @@ export const FormStatus = memo<Props>(({ status }) => {
           </span>
         </div>
       );
+    case 'alreadyExisting':
+      return (
+        <div className={styles.statusContainer}>
+          <span className={clsx(styles.statusIcon, 'icon-error')}></span>
+          <span>
+            Blog o podanym adresie został już dodany do naszej bazy danych. Jeżeli nie pojawiają się
```

Comment:
Czy to się czymś różni od dwóch poprzednich oprócz tekstu błędu? Możemy przecież zrobić zwykły obiekt status http -> tekst błędu